setup.py

```diff
@@ -26,7 +26,7 @@
 setup(name='marionette-wrapper',
       version=PACKAGE_VERSION,
       description='A wrapper for Marionette that increases test legibility',
-      long_description='See https://github.com/mozilla-services/services-test/tree/dev/services-marionette',  # noqa
+      long_description='See https://github.com/mozilla-services/marionette-wrapper',  # noqa
```

Comment:
Wasn't sure if this should be pointing to the old services-test repo or this repo.